job/jobcache.go

```diff
@@ -1,36 +1,86 @@
 package job
 
 import (
+	"os"
+	"os/signal"
 	"sync"
+	"time"
 )
 
-var (
-	AllJobs = &JobCache{
-		Jobs:   make(map[string]*Job),
-		rwLock: sync.Mutex{},
+type JobCache interface {
+	Get(id string) *Job
```

Comment:
@ajvb Comments on any implementation details that should be handled to fulfill the database?